src/test/store/receive-profile.test.js

```diff
@@ -1006,12 +986,10 @@ describe('actions/receive-profile', function () {
       const hash = 'c5e53f9ab6aecef926d4be68c84f2de550e2ac2f';
       const expectedUrl = `https://storage.googleapis.com/profile-store/${hash}`;
       window.fetch
-        .mockImplementationOnce((_) => Promise.resolve(fetch403Response))
-        .mockImplementationOnce((url) =>
-          Promise.resolve(
-            url === expectedUrl ? fetch200Response : fetch403Response
-          )
-        );
+        .getOnce(expectedUrl, 403)
+        .get(expectedUrl, makeProfileSerializable(_getSimpleProfile()), {
+          overwriteRoutes: false,
+        });
```

Comment:
See https://github.com/wheresrhys/fetch-mock/issues/637 for the background on using `{ overwriteRoutes }`. I don't find this very intuitive to be honest. We can also make `overwriteRoutes: false` the default in the setup. I didn't do it yet but if we find the behavior is better for us this is possible.

Reply:
Ah yeah, this looks interesting. Thanks for filing an issue.